packages/babel-jest/package.json

```diff
@@ -13,6 +13,8 @@
   "exports": {
     ".": {
       "types": "./build/index.d.ts",
+      "require": "./build/index.js",
```

Comment:
I've included `require` here so that bundlers/runtimes who understand both can go straight to it - no need to use the ESM wrapper